### PR TITLE
Task update tranzito utils 1.2.0

### PR DESCRIPTION
**Pull request by ahmed-tranzito**
### Description

- Add a dropdown for period_select(filters)
- Update the filters functionality as per the dropdown
- Add Flatpickr package

#### Note:
This needs to be merged so we can update the gem and JS package in [Curbcms](https://github.com/Tranzito-STAP/CurbCMS/pull/106) in order to reflect the new filter changes.

### CODE REVIEW

File: tranzito_utils_js/dist/dropdown_period_selector.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          .attributes['data-period'].value
+        if (selectedPeriod === 'custom') this.initCustomPeriodSelector()
+      })
+    this.enablePeriodSelection()
+  }
+
+  enablePeriodSelection () {
+    document.querySelectorAll('#periodSelectDropdown .dropdown-item').forEach((dropdwonLink) => {
+      dropdwonLink.addEventListener('click', (e) => {
+        if (dropdwonLink.children[0].attributes['data-period'].value === 'custom') e.preventDefault()
```
Comment (willbarrett) [inline tranzito_utils_js/dist/dropdown_period_selector.js]: I think you meant `dropdownLink`, but you have consistently spelled it `dropdwonLink`, so the code will work, and maybe there's a reason 😄 ?

**Review by willbarrett — COMMENTED** [review]
This looks reasonable other than one misspelling. I'm excited to see it in use!

File: tranzito_utils_js/dist/dropdown_period_selector.js (hunk 1 above)
Comment (ahmed-tranzito) [reply tranzito_utils_js/dist/dropdown_period_selector.js]: Ah.. my bad. I misspelled it once and copy pasted it 😄 Sorry for that.

**Review by willbarrett — APPROVED** [review]
Looks reasonable after the refactor too! Looking forward to seeing it in action.

**Comment by willbarrett** [comment]
Down the road, if we need multiple selectors on the page, using IDs may bite us but I'm willing to postpone dealing with that issue until we have the problem.

**Comment by ahmed-tranzito** [comment]
> Down the road, if we need multiple selectors on the page, using IDs may bite us but I'm willing to postpone dealing with that issue until we have the problem.

I'm not sure, I got  your point. Are you talking about the dropdown button id?

**Pull request merged**